CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# Optimizely Csharp SDK Changelog
+
+## 3.3.0
+August 22nd, 2019
+
+### New Features:
+- Added support for event batching via the event processor.
+- Events generated by methods like `activate`, `track`, and `isFeatureEnabled` will be held in a queue until the configured batch size is reached, or the configured flush interval has elapsed. Then, they will be combined into a request and sent to the event dispatcher.
+- To configure event batching, set the `eventBatchSize` and `eventFlushInterval` properties in the `OptimizelyFactory` using `OptimizelyFactory.SetBatchSize(int batchSize)` and `OptimizelyFactory.SetFlushInterval(TimeSpan flushInterval)` and then create `OptimizelyFactory.NewDefaultInstance`.
+- Event batching is enabled by default. `eventBatchSize` defaults to `10`. `eventFlushInterval` defaults to `30000` Milliseconds.
+- Updated the `dispose` method representing the process of closing the instance. When `dispose` is called, any events waiting to be sent as part of a batched event request will be immediately batched and sent to the event dispatcher.
```

Comment:
`will be immediately batched or timed out depending on the timed out interval` @mikeng13 @mjc1283 is this suggestion look OK?

Reply:
Maybe we can just say `immediately batched and sent to event dispatcher`, then describe the timeout functionality in another bullet point.